mne/viz/_3d.py

```diff
@@ -1134,6 +1148,12 @@ def plot_alignment(info=None, trans=None, subject=None, subjects_dir=None,
                               fwd_nn[:, ori, 1],
                               fwd_nn[:, ori, 2],
                               color=color, mode='arrow', scale=1.5e-3)
+    if 'pairs' in fnirs:
+        fnirs_loc = np.array([info['chs'][k]['loc'][3:9] for k in fnirs_picks])
+        logger.info('Plotting %d fnirs pairs' % (fnirs_loc.shape[0]))
+        renderer.tube(origin=fnirs_loc[:, :3],
+                      destination=fnirs_loc[:, 3:])
```

Comment:
It's about being able to plot multiple tubes with only one instruction. I could not make it work with mayavi. This is what I ended up using:

```suggestion
        for loc in range(len(fnirs_loc)):
            x1, y1, z1 = fnirs_loc[loc, :3]
            x2, y2, z2 = fnirs_loc[loc, 3:]
            renderer.tube(origin=np.c_[x1, y1, z1],
                          destination=np.c_[x2, y2, z2])
```

Reply:
@GuillaumeFavelier i have attempted to fix this by modifying the mayavi code to be consistent with PyVista in the commit below.

Reply:
Or would you prefer to address this in plot|allignment as you suggest above?

Reply:
Agreed it's better to make `renderer` support it natively